tests/e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { ClientFunction, Role, Selector, t } from "testcafe";
+import {slugify} from '../config/utilities'
+import * as process from "process";
+
+const loginName = Selector('input.username')
+const loginPassword = Selector('input.password')
+const createArticle = Selector('a').withAttribute('href', '/article/create')
+const metadataButton = Selector('button').child('i.rs-icon-newspaper-o')
+const createButton = Selector('button').child('i.rs-icon-save')
+const publishButton = Selector('button').child('i.rs-icon-cloud-upload')
+
+const closeButton = Selector('.rs-drawer-footer').child('button.rs-btn-primary')
```

Comment:
can we have it like `drawerCloseButton`? to differ it from modals for example?

Reply:
This is only a test implementation. @camilleory has in here tests a better naming strategy.